locales/de.json

```
@@ -559,6 +559,32 @@
 			"wednesday": "Mittwoch"
 		}
 	},
+	"home": {
+		"button": {
+			"aboutTheProject": "Über das Projekt",
+			"loginAsStudentDemo": "als Schüler:in einloggen",
+			"loginAsTeacherDemo": "als Lehrer:in einloggen",
+			"subscribeNewsletter": "Newsletter abonnieren"
+		},
+		"headline": {
+			"HPISchoolCloudBlog": "<span style=\"white-space: nowrap;\">HPI Schul-Cloud</span> Blog",
+			"allNewsOnHPINewsletter": "Alle Neuigkeiten rund um die <span style=\"white-space: nowrap;\">HPI Schul-Cloud</span> bequem in Ihrem Postfach:",
+			"explainationVideo": "Erklärvideo zur HPI Schul-Cloud",
+			"projectLeaders": "Projektdurchführung",
+			"tryDemoVersion": "Jetzt Demoversion ausprobieren",
+			"welcomeToHPISchoolCloud": "Herzlich willkommen bei der <span style=\"white-space: nowrap;\">HPI Schul-Cloud!</span>"
+		},
+		"link": {
+			"moreArticles": "weitere Artikel anzeigen",
+			"openArticle": "Artikel öffnen"
+		},
+		"text": {
+			"BlogIsDeactivated": "Der Blog ist derzeit deaktiviert! Das HPI Schul-Cloud Team kümmert sich darum.",
+			"moreInformation": "Mehr Informationen: ",
```

```suggestion
			"moreInformation": "Mehr Informationen:",
```
I would prefer to add those spaces where the key is used, otherwise we may accidantely delete it on lokalise and will have a problem, where 2 spaces won't hurt, because browsers will render them as one anyway. 
So if you add a space in the code, nobody can break something if he removes or adds them to the language file.